horovod/tensorflow/mpi_ops.cc

```diff
@@ -62,7 +62,11 @@ namespace gpu {
 GpuStreamHandle AsGpuStreamValue(Stream* stream);
 } // namespace stream_executor
 } // namespace gpu
+#if TENSORFLOW_VERSION >= 2010000000
+#include "tensorflow/compiler/xla/stream_executor/stream.h"
```

Comment:
This appears to change the build with TF 2.10 (released 14 days ago), not only with recent `tf-nightly`. Is that intentional?

I think current TensorFlow master would present itself as 2.11.0, judging from https://github.com/tensorflow/tensorflow/blob/9c73ada2531dd8ec2efe45a5faf44199aed9524e/tensorflow/tensorflow.bzl#L64

Reply:
you are right, thanks for that!